### PR TITLE
Admin: Make logo for about modal configureable

### PR DESCRIPTION
**Pull request by dkarnutsch**
To better support white-label solutions

### CODE REVIEW

File: .changeset/sour-brooms-trade.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-admin": major
```
Comment (nsams) [inline .changeset/sour-brooms-trade.md:2]: you must not make major changes in main branch

Comment (dkarnutsch) [reply .changeset/sour-brooms-trade.md]: of course, that was unintentional (and the change is not breaking)

**Review by nsams — DISMISSED** [review]

**Review by johnnyomair — CHANGES_REQUESTED** [review]
I don't think that's enough. The AboutModal isn't used directly by our applications. It is used in `UserHeaderItem`, which is then used in the application (see demo/admin/src/common/MasterHeader.tsx). Therefore you'll need to add this prop to UserHeaderItem as well.

**Comment by dkarnutsch** [comment]
> I don't think that's enough. The AboutModal isn't used directly by our applications. It is used in `UserHeaderItem`, which is then used in the application (see demo/admin/src/common/MasterHeader.tsx). Therefore you'll need to add this prop to UserHeaderItem as well.

Done, Thanks

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**